### DISCUSSION TITLE
Installing "colour-hdri" on Linux fails.

### DISCUSSION THREAD

**Discussion by ai2ys** [colour-science/colour-hdri#10] [Q&A] [answered]
When trying to install `colour-hdri` on Linux (local Ubuntu 20.04 installation as well as Docker image python:3.8-buster) the installation fails.

Installation of the following libraries (dependencies from the requirements.txt) fail:
- recordclass==0.12.0.1    
- matplotlib==3.0.3    
(error message: `... AttributeError: module 'platform' has no attribute 'linux_distribution'`)

Other versions of the libraries can be installed on the systems, e.g.:
- recordclass==0.14.3
- matplotlib==3.3.3

Edit:
I just tried installing older version. The latest version that did work on Linux is v0.1.6.

**Comment by thomasmansencal** [comment, marked as answer]
Hi @ai2ys,

There is actually no strong requirements besides those given in the [pyproject.toml](https://github.com/colour-science/colour-hdri/blob/develop/pyproject.toml) file or the automatically generated [setup.py](https://github.com/colour-science/colour-hdri/blob/develop/setup.py). The `requirements.txt` is also automatically produced but from what my virtual environment at the time of the packaging was or whenever I update it. It should not really be used as a reliable method to install any of our packages.

Cheers,

Thomas
